dialogue-annotations/src/main/java/com/palantir/dialogue/annotations/Request.java

```diff
@@ -95,4 +95,13 @@
 
         Class<? extends ListParamEncoder<?>> encoder() default DefaultListParamEncoder.class;
     }
+
+    /**
+     * Similar to the Feign QueryMap serialization type. Only supports simple string -> string maps.
```

Comment:
Mind updating the docs to describe the updated functionality? I think this has diverged from the feign implementation a bit, but in a way that allows greater flexibility.

Reply:
Done.